docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1118,15 +1118,38 @@ public class MyConfiguration {
 
 The `supports(Class requestContextClass, Class responseClass,
 Class serverTypeClass)` method can be used to determine whether the processor in question handles objects of provided types. If not overridden by the user, it returns `true`.
 
 NOTE: In the preceding method calls, `RC` means `RequestContext` type, `RES` means client response type, and `T` means returned server type.
 
+[[loadbalancer-micrometer-stats-lifecycle]]
+=== Spring Cloud LoadBalancer Statistics
+
+A `LoadBalancerLifecycle` bean that we provide out of the box is `MicrometerStatsLoadBalancerLifecycle` that uses Micrometer to provide statistics for load-balanced calls.
+
+In order to get this bean added to your application context,
+set the value of the `spring.cloud.loadbalancer.stats.micrometer.enabled` to `true` and have a `MeterRegistry` available (for example, by adding https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html[Spring Boot Actuator] to your project).
+
+`MicrometerStatsLoadBalancerLifecycle` registers the following meters in `MeterRegistry`:
+
+* `loadbalancer.requests.active` gauge that allows you to monitor the number of currently active requests for any service instance (service instance data available via tags);
+* `loadbalancer.requests.success` timer that measures the time of execution of any load-balanced requests that have ended in passing a response on to the underlying client;
+* `loadbalancer.requests.failed` timer that measures the time of execution of any load-balanced requests that have ended with an exception;
+* `loadbalancer.requests.discard` counter that measures the number of discarded load-balanced requests, i.e. requests where a service instance to run the request on has not been retrieved by the LoadBalancer.
+
+Additional information regarding the service instances, request data and response data is added to metrics via tags whenever available.
+
+NOTE: For some implementations, such as `BlockingLoadBalancerClient` request and response data might not be available, as we establish generic types from arguments and might not be able to determine the types and read the data.
+
+NOTE: The meters will be registered in the registry when at least one record is added for a given meter.
+
+TIP: You can further configure the behaviour of those metrics (for example, add response[publishing percentiles and histograms]) by https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-metrics-per-meter-properties[adding `MeterFilters`].
```

Comment:
Remove the "u" from "behaviour"

Why are the brackets around "[publishing percentiles and histograms]"? Is it meant to be a link? If so, it cannot work as it stands. If not, you can remove the brackets and have a correct sentence.

Reply:
Yes, missed the link. Will add.